src/utils/RestfulUtils.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Restful Utils
+ * Define common utils used by controllers
+ */
+
+import * as express from "express";
+import { Either, Left, Right } from "fp-ts/lib/Either";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { ControllerError } from "../enums/ControllerError";
+import { HttpErrorStatusCode } from "../enums/HttpErrorStatusCode";
+import { IRestfulObject } from "../types/BaseResponseApp";
+import { FiscalCode } from "../types/FiscalCode";
+import { logger } from "../utils/Logger";
+
+// Utils used by Controllers
+// Send an error message for express response
+export function sendErrorResponse(
```

Comment:
You should probably use [these](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/utils/response.ts) once @gunzip moves them to `italia-ts-commons` (what's the ETA?)

Reply:
anyway, I would not hold this PR further, @LsKoder pls create a technical-debt story for refactoring the code to use those definitions once they lan in `italia-ts-commons`

Reply:
I'm doing it now

Reply:
ref. https://github.com/teamdigitale/italia-ts-commons/pull/33/files